relentless/potential/pair.py

```diff
@@ -589,6 +589,109 @@ class Depletion(PairPotential):
         List of types (A type must be a str).
 
     """
+
+    class LowBound(core.DependentVariable):
```

Comment:
You could cheat and subclass `ArithmeticMean` for this one
```py
class LowBound(core.ArithmeticMean):
    pass
```
to recycle code. The only tradeoff is that the parameters will be named `a` and `b`.